PR: Fix typo in Windows executable filename extensions

StefRe opened the pull request:
## Description of Changes

* fix typo in list of Windows executable filename extensions
* add `.cmd` as alternative to `.bat` files



<!--- Explain what you've done and why --->




### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

<!--- TYPE YOUR USER/NAME AFTER THE FOLLOWING: --->
I certify the above statement is true and correct: @StefRe

<!--- Thanks for your help making Spyder better for everyone! --->

ccordoba12 commented:
Thanks a lot for your contribution! We'd like to include it in our next release in the Spyder 4 series. For that, please make your branch to derive from our `4.x` branch (instead of `master`), with the following commands:

    git checkout 4.x
    git checkout patch-1
    git rebase --onto 4.x master patch-1
    git push -f origin patch-1

If you decline to do that, your work will be part of Spyder 5, to be released at the end of 2020 or in 2021.

StefRe commented:
Thanks for the guidance, I rebased the patch as requested and changed the merge destination to `4.x` (at first I was too quick in switching the base branch, sorry to have messed up the timeline of this PR with that).

ccordoba12 commented:
> at first I was too quick in switching the base branch, sorry to have messed up the timeline of this PR with that

No worries, that's fine.
